.github/workflows/CI.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Add conda to system path
+      run: |
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        echo $CONDA/bin >> $GITHUB_PATH
+    - name: Install dependencies run tests
```

Comment:
May I suggest conda incubator action! https://github.com/marketplace/actions/setup-miniconda

Reply:
can you elaborate? or demonstrate?

Reply:
https://github.com/conda-incubator/setup-miniconda/blob/master/action.yml 
This elaborates more on the options and things you can do with miniconda action. In general, this action helps us setup a conda env more efficiently.

Reply:
can you push a commit(s) to this PR that makes the changes you suggest?.

Reply:
I have a question about MOPAC, is it needed for tests and how is the env variable setup? is it a secret? I updated the CI with miniconda setup. Takes about [~5 min](https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2072/checks?check_run_id=2143247106)s to install conda env from the file

Reply:
Yeah, the mopac key has to be set up for the tests to run, otherwise they're skipped (like the gaussian tests). It was previously defined in .travis.yml.

Here are explanations for all of the secrets that the Travis build has, as listed in the build logs.

Secret environment variables which were configured via the Travis website:
```
# Old keys for triggering RMG-tests, no longer used
$ export encrypted_244ac3091cff_key=[secure]
$ export encrypted_244ac3091cff_iv=[secure]
# GitHub access token for triggering RMG-tests, replaced above keys, set up from RMG Dev user account
$ export GH_TOKEN=[secure]
# Conda push token for automatically building RMG conda binaries and pushing to anaconda.org, set up from RMG organization
$ export CONDA_TOKEN=[secure]
```

Secret environment variables from [.travis.yml](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/.travis.yml#L3), should match the order they're listed in the file
```
# MOPAC key, I don't know what the original value is, but any valid key would work
$ export MOPACKEY=[secure]
# Thermo Central Database connection info and credentials - the server is no longer running, and the unit tests are skipped automatically, so these probably aren't needed anymore
$ export TCD_USER=[secure]  
$ export TCD_PW=[secure]
$ export TCD_HOST=[secure]
$ export TCD_PORT=[secure]
```